gnmi_cli_py/py_gnmicli.py

```diff
@@ -113,6 +113,8 @@ def _create_parser():
                       '\nCan be Leaf value or JSON file. If JSON file, prepend'
                       ' with "@"; eg "@interfaces.json".',
                       required=False)
+  parser.add_argument('--proto', type=str, help='Output files for proto bytes',
+                      nargs="+", required=False)
```

Comment:
>"+" [](http://example.com/codeflow?start=28&length=3)

You assume at least one file. If the query is expected to fail, do you allow gnmicli to have 0 args?

Reply:
updated to use *